docs/README.md

```diff
@@ -1,5 +1,10 @@
 # docs
-Here we document important concepts and functionality formica supports.
+Here we document important concepts and functionality Inago supports.
 
 ### table of contents
-- [slice expansion](slice_expansion.md)
+
+- [What is the goal of Inago?](goal.md)
+- [What do we expect from the users?](expect.md)
+- [What are the terms we use and what do they mean?](terms.md)
+- [What is the structure in fleet we generate?](fleet_structure.md)
```

Comment:
I am not sure what structure this means then. In fleet there are simply units. You define a group based on the unit files within the group directory and then you have this units scheduled to the fleet cluster. I don't know what to document here. Maybe you want to simply write this on your own. Feel free to hijack the PR.